### PR TITLE
chore:  #450 copy types definition into graphql server repo

### PR DESCRIPTION
**Pull request by trankhacvy**
fixes: #450

### CODE REVIEW

File: packages/graphql-server/src/20200131/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (tanphamhaiduong) [inline packages/graphql-server/src/20200131/package.json]: Pls remove it from this folder

File: packages/graphql-server/src/20200131/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Foundations TS Definitions:
```
Comment (tanphamhaiduong) [inline packages/graphql-server/src/20200131/README.md]: Pls remove this

File: packages/graphql-server/src/20200131/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (tanphamhaiduong) [inline packages/graphql-server/src/20200131/tsconfig.json]: Pls remove this

**Comment by trankhacvy** [comment]
> * The task was to create a script to do this automatically
> * Should only import the definitions themselves for platform, we don't need the marketplace ones and all the other manifest files

@willmcvay For now, I just manually copy these files and create this pull request for other people can resolve these graphql pull requests. I will create the command line later.

**Pull request merged**